authentication/subscription/service/oso_subscription_service.go

```diff
@@ -170,10 +170,50 @@ func (s *osoSubscriptionServiceImpl) loadSubscriptions(ctx context.Context, user
 		log.Error(ctx, map[string]interface{}{
 			"err":         err,
 			"reg_app_url": regAppURL,
+			"username":    username,
 			"body":        bodyString,
 		}, "unable to unmarshal json with subscription status")
 		return nil, autherrors.NewInternalError(ctx, err)
 	}
 
 	return &sbs, nil
 }
+
+// DeactivateUser deactivates the user on OpenShift
+func (s *osoSubscriptionServiceImpl) DeactivateUser(ctx context.Context, username string) error {
+	// Load status from OSO
+	regAppURL := fmt.Sprintf("%s/api/accounts/%s/deprovision_osio?authorization_username=%s",
+		s.config.GetOSORegistrationAppURL(), username, s.config.GetOSORegistrationAppAdminUsername())
+	log.Debug(ctx, map[string]interface{}{
+		"url": regAppURL,
```

Comment:
`url` is `reg_app_url` in all other logs.  can we have same name in all logs.

Reply:
ok, let me change that

Reply:
done in d770717